### PR TITLE
Final updates

### PR DESCRIPTION
**Pull request by spzj**
- [x] Update test cases
- [x] Update pattern in `DeletePatientCommandParser` and `DeleteDoctorCommandParser` (Let me know if there are any issues) 
- [x] Replace messages imports with single class static imports
- [x] Refactor find command messages

There were initially merge conflicts but I've resolved them. Let me know if there are any issues.
Currently not passing the Java CI due to the trailing whitespaces in the DG.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/UserInputParser.java [hunk 1]
```diff
@@ -42,6 +41,7 @@
 import seedu.address.logic.parser.patient.AddPatientCommandParser;
 import seedu.address.logic.parser.patient.DeletePatientCommandParser;
 import seedu.address.logic.parser.patient.EditPatientCommandParser;
+import seedu.address.logic.commands.patient.FindPatientCommand;
```
Comment (onnwards) [inline src/main/java/seedu/address/logic/parser/UserInputParser.java]: looks like a checkstyle issue here

File: src/test/java/seedu/address/logic/parser/UserInputParserTest.java [hunk 2]
```diff
@@ -20,9 +20,9 @@
 import seedu.address.logic.commands.patient.DeletePatientCommand;
 import seedu.address.logic.commands.patient.EditPatientCommand;
 import seedu.address.logic.commands.patient.EditPatientCommand.EditPatientDescriptor;
-import seedu.address.logic.commands.patient.FindPatientCommand;
 import seedu.address.logic.commands.patient.ListPatientCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.logic.commands.patient.FindPatientCommand;
```
Comment (onnwards) [inline src/test/java/seedu/address/logic/parser/UserInputParserTest.java]: more checkstyle issues

File: src/test/java/seedu/address/logic/commands/patient/DeletePatientCommandTest.java [hunk 3]
```diff
@@ -27,69 +31,109 @@
...
+        assertCommandFailure(deletePatientCommand, emptyApptScheduleModel, MESSAGE_INVALID_PATIENT_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void execute_forceValidIndexUnfilteredPatientRecordsExistingAppointment_success() {
+        Model nonEmptyApptScheduleModel = new ModelManager(getTypicalPatientRecords(), getTypicalDoctorRecords(),
+                getTypicalAppointmentSchedule(), new UserPrefs());
+
+        Patient patientToDelete = nonEmptyApptScheduleModel.getFilteredPatientList().get(
+                INDEX_SECOND_IN_LIST.getZeroBased());
```
Comment (onnwards) [inline src/test/java/seedu/address/logic/commands/patient/DeletePatientCommandTest.java:120]: wouldn't this be the same as `INDEX_FIRST_IN_LIST.getOneBased()`?

**Review by onnwards — APPROVED** [review]
fix checkstyle and should be good, also, what does the new pattern do?

File: src/test/java/seedu/address/logic/commands/patient/DeletePatientCommandTest.java (hunk 3 above)
Comment (spzj) [reply src/test/java/seedu/address/logic/commands/patient/DeletePatientCommandTest.java:120]: I removed the appointments for Alice a long time ago so I used second in list instead. I'm going to add back Alice's appointment, but this testcase would work still.

File: src/test/java/seedu/address/logic/parser/UserInputParserTest.java (hunk 2 above)
Comment (spzj) [reply src/test/java/seedu/address/logic/parser/UserInputParserTest.java]: Noted, probably surfaced during refactoring.

File: src/main/java/seedu/address/logic/parser/UserInputParser.java (hunk 1 above)
Comment (spzj) [reply src/main/java/seedu/address/logic/parser/UserInputParser.java]: Noted, probably surfaced during refactoring.

**Pull request merged**